client/src/pages/JoinGame.tsx

```diff
@@ -11,6 +13,15 @@ const JoinGame = () => {
     socket.emit("JOIN_GAME", gameCode, playerName);
   };
 
+  const [gameCodeCreated, setGameCode] = useState("");
+  const [player, setPlayer] = useState<PlayerAttributes>();
+  useEffect(() => {
+    socket.on("JOINED_GAME", (player, gameCode) => {
```

Comment:
We don't need to recieve the gameCode from the socket right? We have access to it in the component state.